libcxx/include/__bit_reference

```diff
@@ -453,7 +454,10 @@ private:
   _LIBCPP_HIDE_FROM_ABI
   _LIBCPP_CONSTEXPR_SINCE_CXX20 explicit __bit_iterator(__storage_pointer __s, unsigned __ctz) _NOEXCEPT
       : __seg_(__s),
-        __ctz_(__ctz) {}
+        __ctz_(__ctz) {
+    _LIBCPP_ASSERT_VALID_INPUT_RANGE(
```

Comment:
I think this should be an internal assertion too. This is a private constructor, only called from within libc++ itself.